indico/modules/categories/blueprint.py

```diff
@@ -57,6 +57,8 @@ def _redirect_event_creation(category_id, event_type):
 _bp.add_url_rule('/manage/roles/<int:role_id>/members', 'add_members', RHAddCategoryRoleMembers, methods=('POST',))
 _bp.add_url_rule('/manage/roles/<int:role_id>/members/<int:user_id>', 'remove_member', RHRemoveCategoryRoleMember,
                  methods=('DELETE',))
+_bp.add_url_rule('/manage/roles/<int:role_id>/members/import', 'add_members_import_csv',
+                 RHAddCategoryRoleMembersImportCSV, methods=('GET', 'POST'))
```

Comment:
I think this `RH` name doesn't need the `Add` part. `RHCategoryRolesMembersImportCSV` should be enough

Reply:
Yeah, that's also fine.